common/config/babel/.babelrc.js

```diff
@@ -34,7 +34,9 @@ process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
         // dialpad
         'dialpad',
         // rooms
-        'rooms'
+        'rooms',
+        // Add people dropdown and dialpad modal in people pane
+        'PeoplePaneDropdown',
```

Comment:
nit: Can we rename this to `people-pane-dropdown` to be consistent with other feature flag names?